reactfire/storage/index.tsx

```diff
@@ -50,32 +43,19 @@ export function useStorageTask<T = unknown>(
  * @param ref - reference to the blob you want to download
  * @param options
  */
-export function useStorageDownloadURL<T = string>(
-  ref: storage.Reference,
-  options?: ReactFireOptions<T>
-): string | T {
-  return useObservable(
-    getDownloadURL(ref),
-    `storage:downloadUrl:${ref.toString()}`,
-    options ? options.startWithValue : undefined
-  );
+export function useStorageDownloadURL<T = string>(ref: storage.Reference, options?: ReactFireOptions<T>): string | T {
+  return useObservable(getDownloadURL(ref), `storage:downloadUrl:${ref.toString()}`, options ? options.startWithValue : undefined);
 }
 
 type StorageImageProps = {
   storagePath: string;
   storage?: firebase.storage.Storage;
 };
 
-export function StorageImage(
-  props: StorageImageProps &
-    React.DetailedHTMLProps<
-      React.ImgHTMLAttributes<HTMLImageElement>,
-      HTMLImageElement
-    >
-) {
+export function StorageImage(props: StorageImageProps & React.DetailedHTMLProps<React.ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>) {
   let { storage, storagePath, ...imgProps } = props;
 
-  storage = storage || useFirebaseApp().storage();
+  storage = storage || useStorage();
```

Comment:
this is the relevant change. everything else is just noise from changing our prettier config